src/diffusers/models/attention_processor.py

```diff
@@ -65,13 +65,18 @@ def __init__(
         out_bias: bool = True,
         scale_qk: bool = True,
         only_cross_attention: bool = False,
+        eps: float = 1e-5,
+        rescale_output_factor: float = 1.0,
+        residual_connection: bool = False,
```

Comment:
Configurable group norm epsilon, output rescale, and residual connection

Reply:
At some point (if need be), we should maybe also serialize them in the config.json of UNet. WDYT?

Reply:
if needed :)